docs/user/commercial/privacy-level.rst

```diff
@@ -1,3 +1,5 @@
+.. TODO: This is a super weird page..
```

Comment:
It is. I feel like this should be application UI instead probably.

Reply:
Yea, I've hidden it for now. I wanted to start with all the pages in a TOC, but I've now marked a few as orphans that I don't think we want to put in front of users quite yet.